[TF 2.0 docs] Add description for activation functions

aweers opened the pull request:
Add descriptions to softplus, softsign and tanh. I am not a native English speaker, so maybe there are some spelling errors.

aweers committed:
[TF 2.0 docs] Add description for activation functions

Add descriptions to softplus, softsign and tanh. I am not a native English speaker, so maybe there are some spelling errors.

kyscg commented on tensorflow/python/keras/activations.py:
```diff
@@ -122,7 +121,11 @@ def selu(x):
-  """Softplus activation function.
+  """The Softplus activation function is a smoothed version of
+  the relu function. For negative values the function returns a 
+  value close to zero, for positve values the result of 
```
```suggestion
  value close to zero, for positive values the result of 
```

kyscg commented on tensorflow/python/keras/activations.py:
```diff
@@ -122,7 +121,11 @@ def selu(x):
+  """The Softplus activation function is a smoothed version of
+  the relu function. For negative values the function returns a 
+  value close to zero, for positve values the result of 
+  the function is similiar to the result a linear function 
```
```suggestion
  the function is similar to what a linear function 
```

kyscg commented on tensorflow/python/keras/activations.py:
```diff
@@ -135,7 +138,12 @@ def softplus(x):
 @keras_export('keras.activations.softsign')
 def softsign(x):
-  """Softsign activation function.
+  """The Softsign activation function is similiar to the 
```
```suggestion
  """The Softsign activation function is similar to the 
```

kyscg commented on tensorflow/python/keras/activations.py:
```diff
@@ -135,7 +138,12 @@ def softplus(x):
+  values close to 1 and for small values the function 
+  returns values close to -1. Like tanh the Softsign 
+  activation function goes through the origin, but 
+  has a smaller slope than tanh. 
```
```suggestion
  has a smaller slope. 
```

kyscg commented on tensorflow/python/keras/activations.py:
```diff
@@ -171,7 +179,10 @@ def relu(x, alpha=0., max_value=None, threshold=0):
 @keras_export('keras.activations.tanh')
 def tanh(x):
-  """Hyperbolic Tangent activation function.
+  """The Hyperbolic Tangent activation function is a similiar to the 
```
```suggestion
  """The Hyperbolic Tangent activation function is a similar to the 
```

kyscg commented on tensorflow/python/keras/activations.py:
```diff
@@ -171,7 +179,10 @@ def relu(x, alpha=0., max_value=None, threshold=0):
 def tanh(x):
-  """Hyperbolic Tangent activation function.
+  """The Hyperbolic Tangent activation function is a similiar to the 
+  sigmoid activation function. It has also a smooth curve from its 
```
```suggestion
  sigmoid activation function. It also is a smooth curve ranging within 
```

kyscg commented on tensorflow/python/keras/activations.py:
```diff
@@ -171,7 +179,10 @@ def relu(x, alpha=0., max_value=None, threshold=0):
-  """Hyperbolic Tangent activation function.
+  """The Hyperbolic Tangent activation function is a similiar to the 
+  sigmoid activation function. It has also a smooth curve from its 
+  minimum to its maximum value. The difference to the sigmoid function 
```
```suggestion
  its minimum and maximum value. The difference from the sigmoid function 
```

kyscg reviewed (changes requested):
Nice contribution, could you please make the following changes?

I've made them out for you, just accept the suggested changes.

Cheers!!

kyscg committed:
Update tensorflow/python/keras/activations.py

Co-Authored-By: aweers <32593524+aweers@users.noreply.github.com>

kyscg committed:
Update tensorflow/python/keras/activations.py

Co-Authored-By: aweers <32593524+aweers@users.noreply.github.com>

kyscg committed:
Update tensorflow/python/keras/activations.py

Co-Authored-By: aweers <32593524+aweers@users.noreply.github.com>

kyscg committed:
Update tensorflow/python/keras/activations.py

Co-Authored-By: aweers <32593524+aweers@users.noreply.github.com>

kyscg committed:
Update tensorflow/python/keras/activations.py

Co-Authored-By: aweers <32593524+aweers@users.noreply.github.com>

kyscg committed:
Update tensorflow/python/keras/activations.py

Co-Authored-By: aweers <32593524+aweers@users.noreply.github.com>

kyscg committed:
Update tensorflow/python/keras/activations.py

Co-Authored-By: aweers <32593524+aweers@users.noreply.github.com>

kyscg commented:
We've signed the CLA's. @rthadur, could you set CLA's to yes and review please

Thanks..

aweers commented:
Thanks, @kyscg for the improvements! ;)

kyscg commented:
@rthadur, gentle ping to review and set CLA's to yes

rthadur commented:
@aweers thank you for your contribution , please make sure all the contributors have signed CLA

aweers commented:
@rthadur @kyscg and I signed the cla and we are the only contributors ;)

kyscg commented:
@rthadur @akshaym Gentle ping to review

fchollet reviewed (changes requested):
Thank you for the PR. You should format docstrings according to the TensorFlow style conventions; in particular, the docstring should start with a single line ending in a period, followed by 2 line breaks.

aweers committed:
Change style according to tf guidelines

Starting each docstring with a line ending with a period and followed by to line breaks.

aweers commented:
Hi @fchollet, thanks for your suggestions! I hope now everything matches the style conventions. ;)

mellanox-github commented:
Can one of the admins verify this patch?

rthadur commented:
@aweers can you please resolve conflicts ?

tensorflowbutler commented:
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

tensorflowbutler commented:
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.
